src/pull/index.js

```diff
@@ -65,6 +73,14 @@ export async function pull({
     }
 } = {}) {
     if (!ensureGit(out, progress)) return;
+    /** @type string[] */
+    let excludes = [];
+    try {
+        const config = readConfig(out);
+        excludes = config.template_pull.exclude_file_paths ?? [];
+    } catch (e) {
+        progress.log(2, 'No configuration file', e);
+    }
```

Comment:
Where we're reading the config file on line 110, we could refactor `compareIqgeorc` and parse/set excludes there instead of using `readConfig`.

If not, we could call `readConfig` inside that `if` check and set `excludes` from there instead.

Reply:
thanks

Reply:
Where `readConfig` is now, would it throw if there's no config file in the project? Writing the copied template doesn't happen until the end of the function